src/raftstore/store/peer.rs

```diff
@@ -191,6 +191,11 @@ pub struct Peer {
     pending_reads: ReadIndexQueue,
     // Record the last instant of each peer's heartbeat response.
     pub peer_heartbeats: FlatMap<u64, Instant>,
+
+    /// Record the instants of peers being added into the configuration.
+    /// Remove them after they are not pending any more.
+    pub peer_pending_after: FlatMap<u64, Instant>,
```

Comment:
I believe a `Vec` is enough.

Reply:
seem it has no effect if the TiKV restarts.

Reply:
yes, the cache can't work immediately after restart.